### ISSUE TITLE
\cline in array

### ISSUE THREAD

**Issue by molke-productions** [brucemiller/LaTeXML#1681]
LaTeXML 0.8.6 ends with an error
```
! You can't use `\spacefactor' in math mode.
<recently read> \spacefactor 
                             
l.57 \@
       cline{1-2}\cr 0&e\end{array},\endDISPLAY\clearpage
```
with a `\cline` in array at math image generation:
```tex
\documentclass{article}

\begin{document}
\[
\begin{array}{c|c}
	\hat{g} & \star\\
	\cline{1-2}
	0 & e
\end{array},
\]
\end{document}
```
Without thorough testing or understanding much Perl (or the full implications), it works for me (same output as 0.8.5) if I remove `\noalign`from the definition (no `\cr`) and add `reversion  => '\cline{#1}'` at the end (no `@`).

https://github.com/brucemiller/LaTeXML/blob/be6bbab59dbf48b7834d96db65c49226c27a96b4/lib/LaTeXML/Package/LaTeX.pool.ltxml#L3543-L3557

By the way, is there a good place to override single methods, definitions, … from the package modules without copying the whole module? I’m not that happy with hacking (especially the more general) modules at an update.

**Comment by molke-productions** [comment]
Thanks for the immediate response. Asumming the same principle as in TeX’s `\def` (that is, the later definition overrides) applies to `DefMacro` or `DefConstructor`, I have an idea what to do.
What about `sub`? I’m currently overriding parts of e.g. `addColumnToMathFork` or `equationgroupJoinCols` in TeX.pool.ltxml.

**Comment by brucemiller** [comment]
This seems to be working now, probably fixed by #1895.  Thanks for the report!
